--- origsrc/bind-9.9.2/bin/python/Makefile.in	2012-09-26 19:35:19.000000000 -0500
+++ src/bind-9.9.2/bin/python/Makefile.in	2012-10-10 23:34:13.093094200 -0500
@@ -29,7 +29,7 @@ MANOBJS =	${MANPAGES} ${HTMLPAGES}
 
 @BIND9_MAKE_RULES@
 
-dnssec-checkds: ${srcdir}/dnssec-checkds.py
+dnssec-checkds: dnssec-checkds.py
 
 doc man:: ${MANOBJS}
 
@@ -41,7 +41,7 @@ installdirs:
 	$(SHELL) ${top_srcdir}/mkinstalldirs ${DESTDIR}${mandir}/man8
 
 install:: ${TARGETS} installdirs
-	${INSTALL_PROGRAM} dnssec-checkds@EXEEXT@ ${DESTDIR}${sbindir}
+	${INSTALL_PROGRAM} dnssec-checkds ${DESTDIR}${sbindir}
 	${INSTALL_DATA} ${srcdir}/dnssec-checkds.8 ${DESTDIR}${mandir}/man8
 
 clean distclean::
