--- origsrc/bind-9.9.6/lib/lwres/lwconfig.c	2014-10-12 13:53:08.206641700 -0500
+++ src/bind-9.9.6/lib/lwres/lwconfig.c	2014-10-08 15:57:40.000000000 -0500
@@ -51,6 +51,11 @@
  *    /etc/resolv.conf
  */
 
+#if defined(__CYGWIN__) && !defined(CYGWIN_LWCONFIG_C)
+#define CYGWIN_LWCONFIG_C
+#include "win32/lwconfig.c"
+#else
+
 #include <config.h>
 
 #include <assert.h>
@@ -774,3 +779,5 @@ lwres_conf_get(lwres_context_t *ctx) {
 
 	return (&ctx->confdata);
 }
+
+#endif /* __CYGWIN__ && !CYGWIN_LWCONFIG_C */
--- origsrc/bind-9.9.6/lib/lwres/win32/lwconfig.c	2014-10-12 13:53:08.214642700 -0500
+++ src/bind-9.9.6/lib/lwres/win32/lwconfig.c	2014-10-12 13:35:06.706308600 -0500
@@ -39,7 +39,12 @@
 #include "../lwconfig.c"
 #undef lwres_conf_parse
 
+#ifdef __CYGWIN__
+#include <windows.h>
+#include <iptypes.h>
+#else
 #include <iphlpapi.h>
+#endif
 
 #define TCPIP_SUBKEY	\
 	"SYSTEM\\CurrentControlSet\\Services\\Tcpip\\Parameters"
@@ -95,6 +100,10 @@ lwres_conf_parse(lwres_context_t *ctx, c
 	ULONG    BufLen = sizeof(FIXED_INFO);
 	DWORD    dwRetVal;
 	IP_ADDR_STRING *pIPAddr;
+#ifdef __CYGWIN__
+	static HMODULE hIphlpapi = NULL;
+	static DWORD (WINAPI *GetNetworkParams)(PFIXED_INFO, PULONG) = NULL;
+#endif
 
 	REQUIRE(ctx != NULL);
 	confdata = &ctx->confdata;
@@ -109,6 +118,14 @@ lwres_conf_parse(lwres_context_t *ctx, c
 	/*
 	 * We didn't get any nameservers so we need to do this ourselves
 	 */
+#ifdef __CYGWIN__
+	if (!GetNetworkParams) {
+		hIphlpapi = LoadLibraryEx ("iphlpapi.dll", NULL,
+					   LOAD_LIBRARY_SEARCH_SYSTEM32);
+		GetNetworkParams = (typeof (GetNetworkParams)) GetProcAddress (hIphlpapi, "GetNetworkParams");
+		if (!GetNetworkParams) return (LWRES_R_FAILURE);
+	}
+#endif
 	FixedInfo = (FIXED_INFO *) GlobalAlloc(GPTR, BufLen);
 	dwRetVal = GetNetworkParams(FixedInfo, &BufLen);
 	if (dwRetVal == ERROR_BUFFER_OVERFLOW) {
