--- origsrc/bind-9.9.0/lib/bind9/Makefile.in	2012-02-29 19:01:07.727342700 -0600
+++ src/bind-9.9.0/lib/bind9/Makefile.in	2012-02-29 14:50:51.000000000 -0600
@@ -69,6 +69,7 @@ libbind9.la: ${OBJS} ${ISCCFGDEPLIBS} ${
 	${LIBTOOL_MODE_LINK} \
 		${CC} ${ALL_CFLAGS} ${LDFLAGS} -o libbind9.la -rpath ${libdir} \
 		-version-info ${LIBINTERFACE}:${LIBREVISION}:${LIBAGE} \
+		-no-undefined \
 		${OBJS} ${DNSLIBS} ${ISCCFGLIBS} ${ISCLIBS} ${LIBS}
 
 timestamp: libbind9.@A@
--- origsrc/bind-9.9.0/lib/dns/Makefile.in	2012-02-29 19:01:07.737342700 -0600
+++ src/bind-9.9.0/lib/dns/Makefile.in	2012-02-29 14:50:52.000000000 -0600
@@ -122,6 +122,7 @@ libdns.la: ${OBJS}
 	${LIBTOOL_MODE_LINK} \
 		${CC} ${ALL_CFLAGS} ${LDFLAGS} -o libdns.la -rpath ${libdir} \
 		-version-info ${LIBINTERFACE}:${LIBREVISION}:${LIBAGE} \
+		-no-undefined \
 		${OBJS} ${ISCLIBS} @DNS_CRYPTO_LIBS@ ${LIBS}
 
 timestamp: libdns.@A@
--- origsrc/bind-9.9.0/lib/isc/Makefile.in	2012-02-29 19:01:07.747342700 -0600
+++ src/bind-9.9.0/lib/isc/Makefile.in	2012-02-29 14:50:51.000000000 -0600
@@ -112,6 +112,7 @@ libisc.la: ${OBJS} ${SYMTBLOBJS}
 	${LIBTOOL_MODE_LINK} \
 		${CC} ${ALL_CFLAGS} ${LDFLAGS} -o libisc.la -rpath ${libdir} \
 		-version-info ${LIBINTERFACE}:${LIBREVISION}:${LIBAGE} \
+		-no-undefined \
 		${OBJS} ${SYMTBLOBJS} ${LIBS}
 
 libisc-nosymtbl.la: ${OBJS}
--- origsrc/bind-9.9.0/lib/isccc/Makefile.in	2012-02-29 19:01:07.767342800 -0600
+++ src/bind-9.9.0/lib/isccc/Makefile.in	2012-02-29 14:50:51.000000000 -0600
@@ -71,6 +71,7 @@ libisccc.la: ${OBJS}
 	${LIBTOOL_MODE_LINK} \
 		${CC} ${ALL_CFLAGS} ${LDFLAGS} -o libisccc.la -rpath ${libdir} \
 		-version-info ${LIBINTERFACE}:${LIBREVISION}:${LIBAGE} \
+		-no-undefined \
 		${OBJS} ${ISCLIBS} ${LIBS}
 
 timestamp: libisccc.@A@
--- origsrc/bind-9.9.0/lib/isccfg/Makefile.in	2012-02-29 19:01:07.777342800 -0600
+++ src/bind-9.9.0/lib/isccfg/Makefile.in	2012-02-29 14:50:51.000000000 -0600
@@ -68,6 +68,7 @@ libisccfg.la: ${OBJS}
 	 ${LIBTOOL_MODE_LINK} \
 		${CC} ${ALL_CFLAGS} ${LDFLAGS} -o libisccfg.la -rpath ${libdir} \
 		-version-info ${LIBINTERFACE}:${LIBREVISION}:${LIBAGE} \
+		-no-undefined \
 		${OBJS} ${DNSLIBS} ${ISCCCLIBS} ${ISCLIBS} ${LIBS}
 
 timestamp: libisccfg.@A@
--- origsrc/bind-9.9.0/lib/lwres/Makefile.in	2012-02-29 19:01:07.787342800 -0600
+++ src/bind-9.9.0/lib/lwres/Makefile.in	2012-02-29 14:50:51.000000000 -0600
@@ -69,6 +69,7 @@ liblwres.la: ${OBJS} version.@O@
 	${LIBTOOL_MODE_LINK} \
 		${CC} ${ALL_CFLAGS} ${LDFLAGS} -o liblwres.la -rpath ${libdir} \
 		-version-info ${LIBINTERFACE}:${LIBREVISION}:${LIBAGE} \
+		-no-undefined \
 		${OBJS} version.@O@ ${LIBS}
 
 timestamp: liblwres.@A@
